dhis-2/dhis-services/dhis-service-analytics/src/main/java/org/hisp/dhis/analytics/table/JdbcTrackedEntityAnalyticsTableManager.java

```diff
@@ -526,19 +379,19 @@ public void populateTable(AnalyticsTableUpdateParams params, AnalyticsTableParti
                 sql.append(
                     replaceQualify(
                         """
-                    \s left join ${trackedentityattributevalue} "${teaUid}" on "${teaUid}".trackedentityid=te.trackedentityid \
-                    and "${teaUid}".trackedentityattributeid = ${teaId}""",
+                    \s left join ${trackedentityattributevalue} ${teaUid} on ${teaUid}.trackedentityid=te.trackedentityid \
+                    and ${teaUid}.trackedentityattributeid = ${teaId}""",
                         Map.of(
-                            "teaUid", tea.getUid(),
+                            "teaUid", sqlBuilder.quote(tea.getUid()),
```

Comment:
Can use shorthand `quote` method.

Reply:
I think this was fixed in some other pr? I got a conflict,